api/client/cli.go

```diff
@@ -67,7 +67,13 @@ func (cli *DockerCli) CheckTtyInput(attachStdin, ttyMode bool) error {
 	// be a tty itself: redirecting or piping the client standard input is
 	// incompatible with `docker run -t`, `docker exec -t` or `docker attach`.
 	if ttyMode && attachStdin && !cli.isTerminalIn {
-		return errors.New("cannot enable tty mode on non tty input")
+		eText := "Error: The input device is not a TTY"
```

Comment:
Not sure; should errors have `Error: ` in front, or without (and start with a lowercase)? Think there was some convention for that